src/common/table/__tests__/__snapshots__/Table.test.jsx.snap

```diff
@@ -8,7 +8,7 @@ exports[`Table renders normally 1`] = `
     <div
       class="table primary noMainHeader"
       role="table"
-      style="min-width:450px"
+      style="min-width:0px"
```

Comment:
This makes tables that overflow horizontally now display incorrectly

Reply:
I'll check if I can control it, I got these changes by merely upgrading the library.

Reply:
So this was the culprit; they [changed the logic of calculating `min-width`](https://github.com/tannerlinsley/react-table/pull/2639/files) in `react-table` v `7.6.0`.

Reply:
Therefore, we have to add `minWidth` to the columns that use `width` property. 
I added them to all the tables that have `width` already set.

Reply:
Great 👍 Is `width` now needed for those columns?

Reply:
I tried it out, and it turns out it is. This is kind of messy, but let's investigate a better solution later.